stories/victory-line.stories.js

```diff
@@ -455,3 +456,22 @@ export const Polar = () => {
     </div>
   );
 };
+
+const StyledCurve = styled(Curve)`
+  stroke: blue;
+  stroke-width: 10px;
+  fill: transparent;
+`;
+
+export const DisableInlineStyles = () => {
+  return (
+    <div style={containerStyle}>
+      <VictoryChart>
+        <VictoryLine disableInlineStyles />
+      </VictoryChart>
+      <VictoryChart>
+        <VictoryLine dataComponent={<StyledCurve disableInlineStyles />} />
+      </VictoryChart>
+    </div>
+  );
+};
```

Comment:
This storybook component isn't actually working yet because it pulls the `evaluateStyle` function from the helpers in the latest version of`victory-core`, which won't actually be updated until this PR is merged. I'm not sure if there's a better way to test these changes for now?

Reply:
@beccanelson if you start a dev server (`yarn start`) and leave it running at the same time that you have your storybook open, all of your packages will rebuild on changes, and all the package linking for local changes will be taken care of by lerna. 

Looking more like you would expect? 
<img width="1385" alt="Screen Shot 2021-06-16 at 10 38 26 AM" src="https://user-images.githubusercontent.com/3719995/122266776-02598b00-ce8f-11eb-89cb-bdcaa4f22eb3.png">

Reply:
Yes, thank you!